plugins/http/guest-js/index.ts

```diff
@@ -186,7 +186,7 @@ export async function fetch(
     throw new Error(ERROR_REQUEST_CANCELLED);
   }
 
-  signal?.addEventListener("abort", () => abort);
+  signal?.addEventListener("abort", () => void abort());
```

Comment:
could you explain what the issue is and how does adding void fixes it?

Reply:
It isn't adding void, the original code did not invoke abort when abort signal had aborted. I added void because I saw eslint comment above about unused promise.

Reply:
The original code had
`signal?.addEventListener("abort", () => abort);` where abort function does not get invoked thus leading to no-op behavior. This was probably misinterpreted as `signal?.addEventListener("abort", abort);`, which does invoke the abort function.

Reply:
@amrbashir I have applied the shorthand syntax, removing void.

Reply:
@twlite thanks for explaining, I didn't notice missing `()`